plugins/toolbox-search/src/toolbox_search.ts

```diff
@@ -161,9 +161,45 @@ export class ToolboxSearchCategory extends Blockly.ToolboxCategory {
     this.flyoutItems_ = query ?
         this.blockSearcher.blockTypesMatching(query).map(
             (blockType) => {
+              // check the block if it has dropdowns
```

Comment:
Can you remove the inline comments throughout? I think the code is self-explanatory!